libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -1026,6 +1026,143 @@
     </mix>
     <output name="out" type="vector4" nodename="N_mix_vector4" />
   </nodegraph>
+  
+  <!--
+    <unifiednoise2d>
+    Combined 2d noises for artists.
+  -->
+  <nodegraph name="NG_unifiednoise2d_float" nodedef="ND_unifiednoise2d_float">
+    <range name="N_range" type="float">
+      <input name="in" type="float" nodename="N_switch_type" value="0" />
+      <input name="gamma" type="float" interfacename="outgamma" value="1" />
+      <input name="outlow" type="float" interfacename="outmin" value="0" />
+      <input name="outhigh" type="float" interfacename="outmax" value="1" />
+      <input name="doclamp" type="boolean" interfacename="clampoutput" value="true" />
+      <input name="inlow" type="float" value="0" />
+      <input name="inhigh" type="float" value="1" />
+    </range>
+    <output name="out" type="float" nodename="N_range" />
+    <switch name="N_switch_type" type="float">
+      <input name="in1" type="float" nodename="N_perlin_noise2d" value="0" />
+      <input name="in2" type="float" nodename="N_cellnoise2d" value="0" />
+      <input name="in3" type="float" nodename="N_worleynoise2d" value="0" />
+      <input name="in4" type="float" nodename="N_fractal3d" value="1" />
+      <input name="which" type="integer" interfacename="type" value="0" />
+      <input name="in5" type="float" value="0" />
+    </switch>
+    <noise2d name="N_perlin_noise2d" type="float">
+      <input name="texcoord" type="vector2" nodename="N_apply_cell_jitter" value="0, 0" />
+      <input name="amplitude" type="float" value="0.5" />
+      <input name="pivot" type="float" value="0.5" />
+    </noise2d>
+    <cellnoise2d name="N_cellnoise2d" type="float">
+      <input name="texcoord" type="vector2" nodename="N_apply_cell_jitter" value="0, 0" />
+    </cellnoise2d>
+    <worleynoise2d name="N_worleynoise2d" type="float">
+      <input name="texcoord" type="vector2" nodename="N_apply_offset" value="0, 0" />
```

Comment:
Although these uniform default values won't impact the rendered results, we should likely remove them for clarity, as they're immediately overridden by node connections.  Would it be helpful if we emitted a warning for this situation in Element::validate, so that it's straightforward to address all such cases at once?

Reply:
Thanks. That's probably not a bad idea, to have a warning emitted. But I'll also make a note to update our script that generated the initial output, so node or interface connections don't also get authored values.